src/components/pages/entities/EntityOutputFiles.vue

```diff
@@ -0,0 +1,241 @@
+              <entity-thumbnail
+                class="output-thumbnail"
+                :output-file-id="outputFile.id"
+                :empty-width="60"
+                :width="60"
+              />
+            </td>
+
+            <task-type-cell
+              class="type"
+              :task-type="getTaskType(outputFile)"
+              :production-id="currentProduction.id"
+            />
+            <td class="original-name">
+              {{ outputFile.original_name }}
+            </td>
+            <td class="revision">
+              {{ outputFile.revision }}
+            </td>
+            <td class="extension">
+              {{ outputFile.extension }}
+            </td>
+            <td class="size">
+              {{ renderFileSize(outputFile.file_size) }}
+            </td>
+            <td class="status">
+              {{ outputFile.validation_status }}
+            </td>
+            <people-name-cell
+              class="person"
+              :person="personMap.get(outputFile.person_id)"
+            />
+
+            <td class="download">
+              <a
+                class="button flexrow-item"
+                :href="getDownloadPath(outputFile.id)"
+                :title="$t('playlists.actions.download_file')"
+                v-if="!isCurrentUserArtist"
+              >
+                <download-icon class="icon is-small" />
+              </a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div v-else>
+      {{ $t('entities.output_files.no_output_files') }}
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+
+import { DownloadIcon } from 'vue-feather-icons'
+import { renderFileSize } from '@/lib/render'
+import EntityThumbnail from '@/components/widgets/EntityThumbnail'
+import PeopleNameCell from '@/components/cells/PeopleNameCell'
+import Spinner from '@/components/widgets/Spinner'
+import TaskTypeCell from '@/components/cells/TaskTypeCell'
+
+export default {
+  name: 'entity-output-files',
+  components: {
+    DownloadIcon,
+    EntityThumbnail,
+    PeopleNameCell,
+    Spinner,
+    TaskTypeCell
+  },
+
+  data() {
+    return {
+      isLoading: false,
+      outputFiles: []
+    }
+  },
+
+  props: {
+    entity: {
+      type: Object,
+      default: () => {}
+    }
+  },
+
+  mounted() {
+    if (!this.entity) return
+    this.reset()
+  },
+
+  computed: {
+    ...mapGetters([
+      'currentProduction',
+      'isCurrentUserArtist',
+      'personMap',
+      'taskMap',
+      'taskTypeMap'
+    ])
+  },
+
+  methods: {
+    ...mapActions(['getEntityPreviewFiles']),
+
+    getTaskType(outputFile) {
+      const task = this.taskMap.get(outputFile.task_id)
+      return this.taskTypeMap.get(task.task_type_id)
+    },
+
+    getDownloadPath(outputFileId) {
+      const type = this.isMovie ? 'movies' : 'pictures'
+      return `/api/${type}/originals/output-files/` + `${outputFileId}/download`
```

Comment:
Use a template string down the line:
```suggestion
      return `/api/${type}/originals/output-files/${outputFileId}/download`
```